src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -349,6 +349,20 @@
 					ToolTipService.ToolTip="{x:Bind Commands.RefreshItems.LabelWithHotKey, Mode=OneWay}">
 					<FontIcon FontSize="14" Glyph="{x:Bind Commands.RefreshItems.Glyph.BaseGlyph}" />
 				</Button>
+
+				<Button
+					x:Name="HomeButton"
+					x:FieldModifier="public"
```

Comment:
What's the reason to make this public?

Reply:
I based this on the other navigation buttons, but I'm not sure why they have it though.

Reply:
You can remove public from all.
I think this is a leftover of something.

Reply:
Also there's one more element that marked as public, TabBar's background l. That is marked as so to be assessable from MainPage to set drag regions. But this can also be improved

Reply:
I pushed a commit removing them